### PR TITLE
Removed unused style attr in pi/index.html 

### PR DESCRIPTION
**Pull request by akanbifatimah**
The Pi/index.html threw an error after I cloned it, I checked and saw that the style attribute on line 132 was empty, however when I inspected and checked the console the styling for this element is from infragram.css. I removed the style attribute in pi/index.html and the code worked fine without any error.

### CODE REVIEW

**Comment by TildaDares** [comment]
Great catch @akanbifatimah!! Thank you!

File: pi/index.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    
+   
+    <!-- http-server to run code  -->
```
Comment (TildaDares) [inline pi/index.html:5]: Can you undo these changes since they are not related to the issue? Thanks!

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: pi/index.html (hunk 1 above)
Comment (akanbifatimah) [reply pi/index.html:5]: I am not clear on the changes to undo, do you mean I should return the code to its former state (add the style attribute back)

**Review by TildaDares — APPROVED** [review]
Thanks @akanbifatimah! Great work.

File: pi/index.html (hunk 1 above)
Comment (akanbifatimah) [reply pi/index.html]: I have undone the changes on line 4 & 5, Please can you confirm, thank you @TildaDares
